tests/stub/configuration_hints/scripts/1_second_exceeds.script

```diff
@@ -3,6 +3,10 @@
 
 C: HELLO {"{}": "*"}
 S: SUCCESS {"server": "Neo4j/4.3.1", "hints": {"connection.recv_timeout_seconds": 1}, "connection_id": "bolt-3"}
+{?
+    C: RESET
+    S: SUCCESS {}
+?}
```

Comment:
You could add the directive `A: RESET`, it reduces the size of script.

Reply:
Thanks, will do.